src/libopensc/libopensc.exports

```diff
@@ -133,11 +135,13 @@ sc_make_cache_dir
 sc_mem_alloc_secure
 sc_mem_clear
 sc_mem_reverse
+_sc_match_atr_block
 sc_match_atr_block
 sc_path_print
 sc_path_set
 sc_pin_cmd
 sc_pkcs1_encode
+sc_pkcs1_strip_02_padding
```

Comment:
Applications that are loading the minidriver typically won't use opensc.dll. Please change it to be consistent with the Windows build.